### DISCUSSION TITLE
Assertion failed : release mode in window 10

### DISCUSSION THREAD

**Discussion by mohamedelbachir** [ocornut/imgui#6119] [New Users Build/Link/Run issues ONLY!]
**Version/Branch of Dear ImGui:**
Version: 1.89.1 WIP (18910)
Branch: [master](https://github.com/ocornut/imgui)

**Back-end/Renderer/Compiler/OS**

Back-ends: imgui_impl_sdl+imgui_impl_sdlrenderer
Compiler: __GNUC__=8 (Codeblock 16.01) [32 bits]
Operating System: Window 10

**My Issue/Question:**
My issue is that when i compile in release mode i got the following assertion failed

**Screenshots/Video**
![image](https://user-images.githubusercontent.com/76158313/215067400-4799d4c2-2ff7-46ea-9434-3c25c8d24320.png)

When i press  [**ignorer**] button is a **french** word who means ignore i get this and so on 

![image](https://user-images.githubusercontent.com/76158313/215068309-138f620f-190f-499f-b4e6-3a68e7de5642.png)

![image](https://user-images.githubusercontent.com/76158313/215069195-abd7e69f-9b81-4904-b8d3-24db53bf6cc8.png)

![image](https://user-images.githubusercontent.com/76158313/215069297-b7e91947-c3b4-4eff-8717-5de5ca1b6acb.png)

![image](https://user-images.githubusercontent.com/76158313/215069409-fddcd3f0-e0e2-42e2-be0c-317b66aef8aa.png)

![image](https://user-images.githubusercontent.com/76158313/215069515-24f19229-2135-468b-8411-4e6f8add8959.png)

### And after the prompts finish to apears the program crash
**thank you !**

**Comment by ocornut** [comment]
Hello,
Transferring to discussions.
Would be good to be more specific. Are you compiling one of the unmodified example app? Are you trying to load a specific font?
99% chance this is a bug in your app/code but we can't guess it without more details.

**Reply by mohamedelbachir** [reply]
no, it's not an unmodified program, however in **debug mode** nothing happening all work fine . excuse for my english i am fluent french

**Reply by mohamedelbachir** [reply]
this is the **ouput** of my **program** in **debug mode**

![image](https://user-images.githubusercontent.com/76158313/215072667-15e11d74-29a6-4cf6-a79e-2b092c193d3a.png)

**thank**

**Reply by ocornut** [reply]
Well you are probably feeding incorrect font data, and unfortunately it’s likely a problem your code, we can’t magically guess what it is.

**Reply by ocornut** [reply]
Most often it would happen if you have uninitialized variables, they would be set differently in both builds..

**Reply by mohamedelbachir** [reply]
thank you it's work :) when i use **freetype** for font display
